pyproject.toml

```diff
@@ -1,3 +1,111 @@
+[build-system]
+build-backend = "hatchling.build"
+requires = ["hatchling"]
+
+[project]
+name = "basket-client"
+dynamic = ["version"]
+description = "A Python client for Mozilla's basket service."
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {text = "BSD-3-Clause"}
+keywords = ["mozilla", "basket"]
+authors = [
+  { name = "Michael Kelly" },
+]
+maintainers = [
+  { name = "Paul McLanahan" },
+  { name = "Rob Hudson" },
+]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Environment :: Web Environment",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: BSD License",
+  "Natural Language :: English",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Topic :: Communications",
+  "Topic :: Software Development :: Libraries",
+]
+dependencies = [
+  "requests",
+]
+
+[project.urls]
+Documentation = "https://github.com/mozilla/basket-client#readme"
+Issues = "https://github.com/mozilla/basket-client/issues"
+Source = "https://github.com/mozilla/basket-client"
+
+[tool.hatch.version]
+path = "src/basket/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+include = [
+  "src/basket/*",
+  "LICENSE",
+  "README.rst",
```

Comment:
The sdist should include all the things needed to build the package. So it should include `MANIFEST.in`, `docs/**`, and `requirements/**`. (Or one `*`. I don't know how globbing works here.)

I doubt you're using sdists and I doubt anyone else is using this library so in this context, it's not a big deal. But it's good to think about.

Also, I'm surprised this repo doesn't have a CODE_OF_CONDUCT file. I thought we all had to add them and someone wrote a bot to make sure all the repos had them.